samples/core/src/test/java/io/oasp/gastronomy/restaurant/general/common/RestTestClientBuilder.java

```diff
@@ -17,26 +17,71 @@
  */
 public class RestTestClientBuilder {
 
+  /**
+   * The port to which the server is bound during testing.
+   */
   private int localServerPort;
 
+  /**
+   * The {@code JacksonJsonProvider}
```

Comment:
avoid such documentation as it adds no additional value. The type is already declared by the field.